src/design-system/InputText.tsx

```diff
@@ -68,18 +72,17 @@ const InputText: React.FC<Props> = (props) => {
   return (
     <TextInputContainer>
       <InputLabelAndHelp label={props.labelAbove} labelHelp={props.labelHelp} />
-      <InputWrapper as="div" style={props.style}>
+      <InputWrapper style={props.style}>
         <WrappedInput
           type={props.type}
-          ref={nameInput}
-          value={inputValue ?? ""}
+          inputRef={nameInput}
+          value={props.valueEntered ?? ""}
           headerStyle={!!props.headerStyle}
           placeholder={props.placeholder ?? placeholder}
-          maxLength={props.maxLength}
-          required={props.required}
```

Comment:
why have we lost these?

Reply:
Oh oops, forgot to add a self-comment here! The library doesn't appear to support them. But also, I checked and it doesn't look like we use them either.

Reply:
that seems ....... bad? questionable at best? these are pretty standard HTML input attributes, are they not?

Reply:
Okay, I did some investigating and yeah you're right, that made absolutely no sense, there is no error. Adding these back!

I'm not sure how this happened, but my best guess as to why I thought these needed to be removed was that when I was in the process of figuring out the correct props interface for CustomDebounceInput, these props weren't working because I maybe used the type for a different, non- `<input>` HTML element.

Reply:
To view the updated diff of just the last two commits, you can go [here](https://github.com/Recidiviz/covid19-dashboard/pull/474/files/7a864ed0a17ba06b350de970468746284fd0cfda..4d1f13c0225ba45d952a726d61de614716830eeb).